### PR TITLE
add code of conduct

### PR DESCRIPTION
**Pull request by garrett**
v1.4.1 from https://www.contributor-covenant.org/

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (garrett) [inline CODE_OF_CONDUCT.md]: We need to add a contact email address for violations.

**Comment by stefwalter** [comment]
Does this apply to the entire cockpit-project or just the cockpit-project/cockpit repo? If the former, then should it live on the website or something like that, rather than in the code base?

**Comment by stefwalter** [comment]
Note that our Privacy Policy is on the cockpit-project.org website, and the code of conduct seems similar, no?

https://cockpit-project.org/privacy.html

**Comment by garrett** [comment]
OK. I can move this to the website and we can link to it from the README, just like we do with the privacy policy.

**Pull request closed without merging**